bioconda_utils/cli.py

```diff
@@ -98,6 +103,24 @@ def build(recipe_folder,
     else:
         docker_builder = None
 
+    # handle git range
+    if git_range:
+        p = sp.run(['git', 'diff',
+                          '--relative=recipes',
+                          '--name-only'] + git_range +
+                         [os.path.join(recipe_folder, '*', 'meta.yaml'),
```

Comment:
Maybe, but also a bit less explicit. I don't think it makes much of a difference. I plan to anyway refactor this a bit in the next days.